frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -2,7 +2,7 @@ import { Button, Container, Nav, Navbar, NavDropdown } from "react-bootstrap";
 import { Link } from "react-router-dom";
 import { hasRole } from "main/utils/currentUser";
 import AppNavbarLocalhost from "main/components/Nav/AppNavbarLocalhost"
-
+// a test
```

Comment:
is this comment needed?

Reply:
No, it was a test at the time but now it serves no purpose, I will remove it!